src/main/kotlin/nl/info/zac/app/admin/model/RestZaakafhandelParameters.kt

```diff
@@ -46,6 +46,10 @@ data class RestZaakafhandelParameters(
      * The frontend currently requires this field to be non-null
      */
     var mailtemplateKoppelingen: List<RESTMailtemplateKoppeling> = emptyList(),
+    /**
+     * The frontend currently requires this field to be non-null
+     */
+    var betrokkeneKoppelingen: RestBetrokkeneKoppelingen? = RestBetrokkeneKoppelingen(),
```

Comment:
```suggestion
    var betrokkeneKoppelingen: RestBetrokkeneKoppelingen = RestBetrokkeneKoppelingen(),
```

If it is indeed non-null as you say (which is a shame) then it should not be nullable right?